java/cuvs-java/src/main/java22/com/nvidia/cuvs/internal/CuVSHostMatrixImpl.java

```diff
@@ -162,12 +214,13 @@ public void close() {}
 
   @Override
   public int get(int row, int col) {
-    return (int) accessor$vh.get(memorySegment, 0L, (long) row * columns + col);
+    return (int) accessor$vh.get(memorySegment, 0L, (long) row * rowStride + col);
```

Comment:
The value of `rowStride` could be `-1` at this point, right?  Shouldn't we only use `rowStride` if it's been set to a valid value?